eventmesh-common/src/test/java/org/apache/eventmesh/common/protocol/http/WebhookProtocolTransportObjectTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.common.protocol.http;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class WebhookProtocolTransportObjectTest {
+
+    WebhookProtocolTransportObject webhookProtocolTransportObject = new WebhookProtocolTransportObject("cloudEventId", "eventType",
+        "cloudEventName", "cloudEventSource", "dataContentType", new byte[] {(byte) 0});
+
+    @Test
+    public void testSetCloudEventId() {
+        webhookProtocolTransportObject.setCloudEventId("cloudEventId");
+    }
+
+    @Test
+    public void testSetEventType() {
+        webhookProtocolTransportObject.setEventType("eventType");
+    }
+
+    @Test
+    public void testSetCloudEventName() {
+        webhookProtocolTransportObject.setCloudEventName("cloudEventName");
+    }
+
+    @Test
+    public void testSetCloudEventSource() {
+        webhookProtocolTransportObject.setCloudEventSource("cloudEventSource");
+    }
+
+    @Test
+    public void testSetDataContentType() {
+        webhookProtocolTransportObject.setDataContentType("dataContentType");
+    }
+
+    @Test
+    public void testSetBody() {
+        webhookProtocolTransportObject.setBody(new byte[]{(byte) 0});
```

Comment:
Body bytes can be found in debug mode.

Reply:
Corrected.

Reply:
Excuse me, I didn't see any change on `testSetBody`. The `cloudEventId` should be `d0b29520-2bba-11ee-877b-2b18ac132e64`, and cloudEventSource and more should also be modified. Maybe running debug mode once would be a good idea. 😊

Reply:
Corrected.
Thank you for pointing out the mistake, I thought fixing this would be enough.
```java
@Before
public void setUp() {
    webhookProtocolTransportObject = WebhookProtocolTransportObject.builder().build();
}
```

Reply:
Not enough. The builder should be used like this:

```
WebhookProtocolTransportObject webhookProtocolTransportObject = WebhookProtocolTransportObject.builder()
            .cloudEventId(cloudEventId).eventType(eventType).cloudEventName(webHookConfig.getCloudEventName())
            .cloudEventSource(webHookConfig.getManufacturerDomain())
            .dataContentType(webHookConfig.getDataContentType()).body(body).build();
```

Pleas substitute getters with expected value. Otherwise, the test will fail.

Moreover, as is said above twice, I suggest give full bytes to `body` in `testSetBody`, instead of `0`, as is shown in the screenshot.

![image](https://github.com/apache/eventmesh/assets/41445332/7203db8f-d9de-4fd2-80d2-6269d19c600a)

You may build a byte array with a test string. As is used in `WebHookController`:

```
ByteBuf buf = ((FullHttpRequest) httpRequest).content();
            webHookController.execute(httpRequest.uri(), header, ByteBufUtil.getBytes(buf));
```

Reply:
Not enough. The builder should be used like this. Otherwise, the test will fail.

```
WebhookProtocolTransportObject webhookProtocolTransportObject = WebhookProtocolTransportObject.builder()
.cloudEventId(cloudEventId).eventType(eventType).cloudEventName(webHookConfig.getCloudEventName())
.cloudEventSource(webHookConfig.getManufacturerDomain())
.dataContentType(webHookConfig.getDataContentType()).body(body).build();
```

Pleas substitute getters with expected value.

Moreover, as is said above twice, I suggest give full bytes to `body` in `testSetBody`, instead of `0`, as is shown in the screenshot.

![image](https://github.com/apache/eventmesh/assets/41445332/7203db8f-d9de-4fd2-80d2-6269d19c600a)

You may build a byte array with a test string. As is used in `WebHookController`:

```
ByteBuf buf = ((FullHttpRequest) httpRequest).content();
webHookController.execute(httpRequest.uri(), header, ByteBufUtil.getBytes(buf));
```

Reply:
> Not enough. The builder should be used like this. Otherwise, the test will fail.
> 
> ```
> WebhookProtocolTransportObject webhookProtocolTransportObject = WebhookProtocolTransportObject.builder()
> .cloudEventId(cloudEventId).eventType(eventType).cloudEventName(webHookConfig.getCloudEventName())
> .cloudEventSource(webHookConfig.getManufacturerDomain())
> .dataContentType(webHookConfig.getDataContentType()).body(body).build();
> ```
> 
> Pleas substitute getters with expected value.
> 
> Moreover, as is said above twice, I suggest give full bytes to `body` in `testSetBody`, instead of `0`, as is shown in the screenshot.
> 
> ![image](https://user-images.githubusercontent.com/41445332/258748769-7203db8f-d9de-4fd2-80d2-6269d19c600a.png)
> 
> You may build a byte array with a test string. As is used in `WebHookController`:
> 
> ```
> ByteBuf buf = ((FullHttpRequest) httpRequest).content();
> webHookController.execute(httpRequest.uri(), header, ByteBufUtil.getBytes(buf));
> ```

this builder just create an instance of an empty attribute.
and set body Corrected.


```java
@Before
public void setUp() {
    webhookProtocolTransportObject = WebhookProtocolTransportObject.builder().build();
}
```